transcendence-app/src/user/decorators/api-file.decorator.ts

```diff
@@ -0,0 +1,17 @@
+import { ApiBody } from '@nestjs/swagger';
+
+export const ApiFile =
+  (fileName = 'file'): MethodDecorator =>
+  (target, propertyKey, descriptor) => {
+    ApiBody({
+      schema: {
+        type: 'object',
+        properties: {
+          [fileName]: {
```

Comment:
I like this way of mapping objects uniquely to data. I wonder if we have more use cases for this.